flocker/node/agents/functional/test_ebs.py

```diff
@@ -315,11 +312,19 @@ class EBSProfiledBlockDeviceAPIInterfaceTests(
     pass
 
 
+class VolumeStub(object):
```

Comment:
This at least allows the state table tests to call the real implemented methods on stub objects that can have their state manipulated.